### PR TITLE
github: add template and action

### PR DESCRIPTION
**Pull request by 0xPoe**

### CODE REVIEW

**Comment by 0xPoe** [comment]
/assign @Mini256 

PTAL~ Thanks!

File: README.md [hunk 1]
```diff
@@ -135,8 +135,8 @@ pull requests on a regular basis. If a core team member believes an RFC PR is re
 
 **TiDB Community Infra's RFC process owes its inspiration to the [React RFC process], [Rust RFC process], [Vue RFC process] and [Ember RFC process]**
 
-[React RFC process]: https://github.com/reactjs/rfcs
-[Rust RFC process]: https://github.com/rust-lang/rfcs
-[Ember RFC process]: https://github.com/emberjs/rfcs
-[Vue RFC process]: https://github.com/vuejs/rfcs
+[react rfc process]: https://github.com/reactjs/rfcs
```
Comment (Mini256) [inline README.md:138]: Why change it to all lowercase? This is inconsistent with the reference above.

Comment (0xPoe) [reply README.md:138]: Prettier done it. It's working.

See: https://github.com/ti-community-infra/rfcs/blob/f35acb207f236547328d339b0856a2d7e7bfb1cb/README.md

**Review by Mini256 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Mini256) [reply README.md:138]: ok.

**Pull request merged**